client/src/app/pages/questionnaires/questionnaires.tsx

```diff
@@ -0,0 +1,459 @@
+import * as React from "react";
```

Comment:
So all components are lower case file names now?

Reply:
You're right since it's a TSX file I suppose it should be Questionnaires.tsx.